index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> 
+	<script src="https://kit.fontawesome.com/ad0c4e912f.js" crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="style.css">
+	<title>Racheal's MintSignup-Clone page</title>
+</head>
+<body>
+	<header>
+	<nav class="headnav">
+		<div class="divbtn">
+			<button  class="bluebtn , nav-submit" > 
+				<span class="white-padlock"><i class="fas fa-lock"></i></span>
+				Sign In
+			</button>
+		</div>
+		<div class="navdiv-qns">
+			<div>Already have an Intuit account?</div>
+			<a id="nav-link" href="#">I forgot my User ID or Password</a>
+		</div>
+	</nav>
+  </header>
+	<main>
+		<div class="intuit"><img src="signup-images/iintuitblack.png" alt="intuit logo"></div>
+		<div class="threelogos">
+			<span><img src="signup-images/mint.png" alt="mint logo"></span> 
+			<p class="imgtag">mint</p>
+	    <span><img src="signup-images/quickbooks.png" alt="quick books logo"></span> 
+			<p class="imgtag">quickbooks</p>
+		  <span><img src="signup-images/turbotax.png" alt="turbotax logo"> </span> 
+			<p class="imgtag">turbotax </p>
```

Comment:
- [ ] I would recommend here using `background images` instead of `img` tag as used in the [Intuit account](https://accounts.intuit.com/signup.html). It is because

- Background images are easy to display properly even at constrained proportions. ...
- Overlaying content over the top of a background image is very easy as you can simply treat it as if there was no image there so you can just apply regular positioning to any elements over the background image.